splunk_setup/splunk.sh

```diff
@@ -0,0 +1,527 @@
+#!/bin/bash
+# Usage: ./splunk.sh <option> <forward-server-ip>
+# Use `indexer` as the forward-server-ip to install the indexer
+
+# Prints script options
+function print_options {
+    echo
+    echo "Usage: ./splunk.sh <option> <forward-server-ip>"
+    echo "Use \`indexer\` as the forward-server-ip to install the indexer"
+    echo "OPTIONS: 
+    -> debian
+    -> linux (general tgz file)
+    -> rpm (red hat distros)
+    -> other (shows list of other forwarder urls)
+    -> -p (prints the specified url debian, linux or rpm in case something is not working)
+            " # trust the indents
+    exit 1
+}
+
+if [ "$#" != 2 ]; then
+    print_options
+fi
+
+###################### DOWNLOAD URLS ######################
+IP="$2"
```

Comment:
These would be great to have in a JSON file so we don't have to update the code.